### PR TITLE
Prepare Release 2.2

### PR DESCRIPTION
**Pull request by danielwilms**
[vss-tools] Update to d5a2e70 Includes the following commits:
* [vspec2ttl] Add dependencies, align with newest rdflib version (d5a2e70)
* Change from GENIVI to COVESA (c506409)
* vspec2yaml: Add --no-uuid option (46c9e1d)
* Adding support for units checking as metadata (07f04ba)
* Removing unix timestamp (959190d)
* minor comment update (22d9cea)
* Optionally allow to not generate UUIDs in output JSON (fe8bf9a)
* Remove old getopt wrapper, using standard argparse (50fc12b)
* README.md: Restore removal of folder structure and tests (d5f5d06)

Update Version file

### CODE REVIEW

**Comment by erikbosch** [comment]
Should we also update the README: https://github.com/danielwilms/vehicle_signal_specification/blob/release2.2/README.md?plain=1#L17

Currently stating:

**Latest release:** [2.1](https://github.com/COVESA/vehicle_signal_specification/releases/tag/v2.1)

Or is that to be changed first after the release, at the same time as we update VERSION to 3.0-develop?

**Comment by danielwilms** [comment]
> Should we also update the README: https://github.com/danielwilms/vehicle_signal_specification/blob/release2.2/README.md?plain=1#L17
> 
> Currently stating:
> 
> **Latest release:** [2.1](https://github.com/COVESA/vehicle_signal_specification/releases/tag/v2.1)
> 
> Or is that to be changed first after the release, at the same time as we update VERSION to 3.0-develop?

I change it to a link to the github releases, to avoid mismatch

**Review by erikbosch — APPROVED** [review]
Looks good to me

**Comment by erikbosch** [comment]
As we update submodules in this PR, should we possibly also update https://github.com/danielwilms/vehicle_signal_specification/blob/release2.2/.gitmodules so that it refer to COVESA and not GENIVI? Maybe good to do it before the release

**Comment by danielwilms** [comment]
> As we update submodules in this PR, should we possibly also update https://github.com/danielwilms/vehicle_signal_specification/blob/release2.2/.gitmodules so that it refer to COVESA and not GENIVI? Maybe good to do it before the release

Good catch. I'll add it to #362

**Review by SebastianSchildt — APPROVED** [review]
Looking good to me.

File: README.md [hunk 1]
```diff
@@ -14,7 +14,7 @@ Please find the official documentation at: [Vehicle Signal Specification](https:
 To use a specific version of VSS in your toolchain, head over to our [releases page](https://github.com/COVESA/vehicle_signal_specification/releases/).
 
 
-**Latest release:** [2.1](https://github.com/COVESA/vehicle_signal_specification/releases/tag/v2.1)
+[Latest release](https://github.com/COVESA/vehicle_signal_specification/releases/latest)
```
Comment (erikbosch) [inline README.md:17]: Change line 24 below as well. Maybe rephrase it to be more generic so we 

Now:

`Work towards version 2.2 is continuously ongoing`

Easier to maintain:

`Work towards next version is continuously ongoing`

Comment (danielwilms) [reply README.md:17]: good idea! changed it @erikbosch
